src/cmd/linuxkit/pkglib/build.go

```diff
@@ -248,34 +253,23 @@ func (p Pkg) Build(bos ...BuildOpt) error {
 		// check local cache, fallback to check registry / pull image from registry, fallback to build
 		fmt.Fprintf(writer, "checking for %s in local cache...\n", ref)
 		for _, platform := range bo.platforms {
-			if exists, err := c.ImageInCache(&ref, "", platform.Architecture); err == nil && exists {
+			exists, err := c.ImageInCache(&ref, "", platform.Architecture)
+			switch {
+			case err != nil && exists:
```

Comment:
Should it be `case err == nil && exists:` as before?

Reply:
Yeah, just maybe.

Reply:
Fixed. Thanks for finding.